src/plugins/timelist/Timelist.vue

```diff
@@ -192,8 +197,8 @@ export default {
             }
         },
         updateTimestamp(_bounds, isTick) {
-            if (isTick === true) {
-                this.timestamp = this.openmct.time.clock().currentValue();
+            if (isTick === true && this.openmct.time.clock() !== undefined) {
```

Comment:
@akhenry Do you think we should debounce this?

Reply:
Yeah, good call. Any high frequency view updates should generally be debounced.